server/src/test/java/org/opensearch/action/bulk/TransportBulkActionIngestTests.java

```diff
@@ -170,7 +171,12 @@ class TestTransportBulkAction extends TransportBulkAction {
                 ),
                 new IndexingPressureService(
                     SETTINGS,
-                    new ClusterService(SETTINGS, new ClusterSettings(SETTINGS, ClusterSettings.BUILT_IN_CLUSTER_SETTINGS), null)
+                    new ClusterService(
+                        SETTINGS,
+                        new ClusterSettings(SETTINGS, ClusterSettings.BUILT_IN_CLUSTER_SETTINGS),
+                        null,
+                        new NoopMetricsRegistryFactory().getMetricsRegistry()
```

Comment:
Replace with NoopMetricsRegistry.INSTANCE

Reply:
Updated in the revision.